Dockerfile

```diff
@@ -1,13 +1,28 @@
 ARG BASEIMAGE
+ARG GOVERSION
 
-FROM $BASEIMAGE AS final 
-COPY ./bin/vg-snapshotter .
+
+FROM golang:${GOVERSION} as builder
+RUN mkdir -p /go/src
+COPY ./ /go/src/
+WORKDIR /go/src/
+RUN CGO_ENABLED=0 \
+    make build
+
+# Final Stage
+FROM $BASEIMAGE AS final
+
+# Copy the binary from the builder stage
+
+COPY --from=builder /go/src/bin/vg-snapshotter .
+
+# Set entry point
 ENTRYPOINT ["/vg-snapshotter"]
 
+# Metadata labels
 LABEL vendor="Dell Inc." \
       name="dellcsi-vg-snapshotter" \
-      summary="CSI VG Snapshotter for Dell EMC Powerflex" \
+      summary="CSI VG Snapshotter for Dell EMC PowerFlex" \
```

Comment:
@KshitijaKakde Since this is being used by both driver i.e. powerstore and powerflex, Can we add PoweFlex/PowerStore?

Reply:
Done